### DISCUSSION TITLE
Integrate SLZB-06 into ioBroker via zigbee2mqtt (help please)

### DISCUSSION THREAD

**Discussion by PaulEins** [Koenkk/zigbee2mqtt#24380] [Devices]
Hello dear community,

Unfortunately I don't know what to do.

I have set up a new ioBroker instance on my Synology. I then installed the zigbee2mqtt adapter there. Unfortunately, I do not know how to connect the Zigbee antenna SLZB-06 to this adapter. :-(

This is what it looks like, maybe one of you understands it. I haven't found anything on the internet.
<img width="1466" alt="Bildschirmfoto 2024-10-17 um 17 41 52" src="https://github.com/user-attachments/assets/7b53661b-50a2-4ca2-a995-2a760d04a46a">
<img width="1058" alt="Bildschirmfoto 2024-10-17 um 17 42 25" src="https://github.com/user-attachments/assets/e826de0b-40ea-4c05-bffc-81cd32d66c2e">
<img width="1283" alt="Bildschirmfoto 2024-10-17 um 17 42 43" src="https://github.com/user-attachments/assets/6f9ba765-ed12-4065-b21a-551984cd6c53">
<img width="1288" alt="Bildschirmfoto 2024-10-17 um 17 42 51" src="https://github.com/user-attachments/assets/fbe58ba5-d5df-4fe7-9edd-935c2903c828">
<img width="1286" alt="Bildschirmfoto 2024-10-17 um 17 42 59" src="https://github.com/user-attachments/assets/9ed76a46-6973-4b96-9c46-ca05aa404ac2">
<img width="1288" alt="Bildschirmfoto 2024-10-17 um 17 43 06" src="https://github.com/user-attachments/assets/05922f9a-2d9f-4e23-8cb6-5b850155d5d7">

Do I have to install another adapter with ioBroker? 

<img width="1284" alt="Bildschirmfoto 2024-10-17 um 21 14 49" src="https://github.com/user-attachments/assets/c086ea12-99f3-46fc-8b65-b0d81112640a">

I can't get to the adapter view either. Could it be that I have to redirect a port in the Docker?
<img width="1470" alt="Bildschirmfoto 2024-10-17 um 18 16 22" src="https://github.com/user-attachments/assets/170d413b-b1c0-4883-8d55-4018f987a863">

If you have any suggestions, I would also be pleased to receive them.
Unfortunately, I don't understand a lot of things and my English is not so good. That's why pure manuals or wikis don't help me. 

I would be very happy if you could help me. Because I don't know what to do and haven't found anything on the internet. I know that I'm not that good. Amateur =) But I'll do my best.

**Comment by rblacky** [comment]
Hallo PaulEins,

I have the same equipment and of course the same problem. Have you found a solution?
Thanks for your answer.
Richard
